h2o-docs/src/product/data-science/algo-params/lambda_search.rst

```diff
@@ -0,0 +1,90 @@
+``lambda_search``
+-----------------
+
+- Available in: GLM
+- Hyperparameter: no
+
+Description
+~~~~~~~~~~~
+
+Setting ``lambda_search`` to TRUE enables efficient and automatic search for the optimal value of the ``lambda`` parameter. When enabled, GLM will first fit a model with maximum regularization (highest lambda value) and then keep decreasing it at each step until overfitting occurs. The resulting model is based on the best lambda value. 
+
+When looking for a sparse solution (``alpha`` > 0), lambda search can also be used to effeciently handle very wide datasets because it can filter out inactive predictors (noise) and only build models for a small subset of predictors. A common use of lambda search is to run it on a dataset with many predictors but limit the number of active predictors to a relatively small value. 
```

Comment:
I am not sure how common use case it is. I would say possible use case. 
The standard use case should be the default - which s let the data determine what is the optimal lambda (and hence set of active predictors)